challenge.md

```diff
@@ -31,21 +31,15 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+This is an absolute path because it contains the root followed by the user. A relative path would look something like this: /directoryexample/subdirectoryexample
```

Comment:
You're right about the path being an absolute path, but the example of a relative path that you give here is wrong. The path that you've written indicates that 'directoryexample' is found inside the root directory. A relative path would be 
```bash
  directoryexample/subdirectoryexample
```
This path tells us only that `directoryexample` is in our current directory _wherever that may be_. Depending on where we are in our filesystem, that relative path may _or may not_ be accurate.